client/extensions/woocommerce/app/products/products-list.js

```diff
@@ -16,11 +16,14 @@ import Button from 'components/button';
 import EmptyContent from 'components/empty-content';
 import { getLink } from 'woocommerce/lib/nav-utils';
 import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
-import { getTotalProducts, areProductsLoaded } from 'woocommerce/state/sites/products/selectors';
 import {
-	getProductListCurrentPage,
-	getProductListProducts,
-	getProductListRequestedPage,
+	getTotalProducts,
+	areProductsLoaded,
+	getProducts,
+} from 'woocommerce/state/sites/products/selectors';
+import {
+	getProductsCurrentPage,
+	getProductsRequestedPage,
 } from 'woocommerce/state/ui/products/selectors';
```

Comment:
I'm a little surprised you didn't move the selectors to `woocommerce/state/selectors` while you were at it. 😄

Reply:
Ha, I do think from a dev-ex point of view that makes more sense (the one-selector-per-file format of calypso's [`state/selectors`](https://github.com/Automattic/wp-calypso/tree/master/client/state/selectors)), but I'm put off for now by the [comment about how this might bloat the build](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/index.js#L10-L14), because we don't have [`transform-imports` configured for extensions](https://github.com/Automattic/wp-calypso/blob/master/.babelrc#L23-L29).

We could set it up so that we're doing things like 

```js
    import getProducts from 'woocommerce/state/selectors/get-products'
```

but that seems like a harder sell 😆